### DISCUSSION TITLE
Question about bunker maps

### DISCUSSION THREAD

**Discussion by MrPigZ** [HelixNebulaStudio/RiseOfTheDead#217] [General]
Currently, maps with a good seed for farming crates are being sold for thousands. If any changes would be made to bunker map generation, for example a new room, how would it affect these preexisting maps?

**Comment by MXKhronos** [comment]
An additional room would likely alter the map's layout exponentially, but I have ideas on how to handle that if were to add new rooms.
I could partition the range of seeds for older maps currently (1-9999). Newer map drops could be from (10000-19999) which contains new rooms.

But the most likely scenario would be that there won't be new rooms added to the map and efforts for a new raid map would be prefered.
